Kudu.Services.Web/Content/Scripts/FileBrowser.js

```diff
@@ -41,7 +41,64 @@ var statusbar = {
         }
 }
 
+var copyProgressObjects = {};
+var copyObjectsManager = {
+    init: function() {
+        copyProgressObjects = {};
+    },
+    addCopyStats: function (uri, loadedData, totalData
+   )  { 
```

Comment:
do you mean that the code should be: 
`addCopyStats: function (uri, loadedData, totalData)  {` 

i.e. having `) {` on the line above?